### PR TITLE
libgig 3.3.0 (new formula)

### PR DESCRIPTION
**Pull request by david0**

### CODE REVIEW

File: Library/Formula/libgig.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class Libgig < Formula
+  desc "library for Gigasampler and DLS (Downloadable Sounds) Level 1/2 files"
+  homepage "http://www.linuxsampler.org/libgig/"
```
Comment (bfontaine) [inline Library/Formula/libgig.rb]: You can use `https` here.

File: Library/Formula/libgig.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  url "http://download.linuxsampler.org/packages/libgig-3.3.0.tar.bz2"
+  sha256 "d22a3c7ba13d920c1d4b6c218107ad105622ae9d1236ffbce007b98547774425"
+
+  depends_on "pkg-config" => :build
+  depends_on "libsndfile"
+
+  def install
+    ENV.deparallelize
```
Comment (bfontaine) [inline Library/Formula/libgig.rb]: Is upstream aware of that?

Comment (david0) [inline Library/Formula/libgig.rb]: Yes, its fixed in upstream version 4.0.0 (which has different build problems):
```
Version 4.0.0 (14 Jul 2015)
  * general changes:
    - minor Makefile fix for parallel make
    - Mac OS X: link with CoreFoundation (for the UUID function)
```

Comment (mikemcquaid) [inline Library/Formula/libgig.rb]: Can you add a comment noting that? I'd also probably rather we used the latest version here.

Comment (david0) [inline Library/Formula/libgig.rb]: I've added comments for that. Yes, the latest version would be probably be better, but for me this is currently sufficient. Macports also hasn't 4.0.0 running and I want avoid being the guinea pig here.

**Comment by apjanke** [comment]
Build is failing at the download stage. We can't use HTTPS because their certificate setup is bad. From Firefox:

```
download.linuxsampler.org uses an invalid security certificate.

The certificate is only valid for the following names:
  bb.linuxsampler.org, linuxsampler.org  

(Error code: ssl_error_bad_cert_domain)
```

Could you report this upstream?

**Comment by david0** [comment]
Sorry, my fault. I updated the formula once again. The homepage can be https, but not the download. Their download links are also http only, so I don't think that this is an issue. They just don't support https for the download subdomain...

**Comment by apjanke** [comment]
It's not you, it's them. Their server is in fact serving up HTTPS traffic on the "download." subdomain; it's just using an invalid certificate to do so. Maybe a web host config issue with respect to virtual hosts or something. I've submitted an upstream bug report: https://bugs.linuxsampler.org/cgi-bin/show_bug.cgi?id=247. If the download is working for you, maybe you said "yes" to a "Should I trust this unverifiable certificate?" prompt at some point.

At any rate, your revision is a good way to handle it, I've merged this PR. Thank you for your contribution to Homebrew!

**Comment by apjanke** [comment]
linuxsampler.com got their HTTPS certificate fixed. I updated the formula to switch back to https.

**Pull request closed without merging**